frontend/src/components/Utils/time.js

```diff
@@ -77,9 +77,30 @@ export function getTimestampFromDateString(dateStr) {
 /**
  * Formats a Firestore timestamp into a date string in ISO format.
  *
+ * No tests for this function because there is no logic.
```

Comment:
Nit: I don't think it's necessary to state whether or not tests for a function exists? I suppose stating it does no harm either but maybe Hiram or Longjun can say more about it.

Reply:
Yeah I agree this is a little silly. I replaced these comments with some TODOs to remove them when integrating timezones anyway.